test/images/agnhost/netexec/netexec.go

```diff
@@ -642,3 +649,22 @@ func assertNoError(err error, detail string) {
 		log.Fatalf("Error occurred: %s:%v", detail, err)
 	}
 }
+
+func parseAddresses(addresses string) []string {
+	if addresses == "" {
+		return []string{bindToAny}
+	}
+	// Using a set to remove duplicates
+	res := make([]string, 0)
+	split := strings.Split(addresses, ",")
+	for _, address := range split {
+		netAddr := net.ParseIP(address)
+		if netAddr == nil {
+			log.Printf("Invalid address %s", address)
+			continue
+		}
+		res = append(res, address)
+	}
+	set := sets.NewString(res...)
+	return set.List()
```

Comment:
what happens if all addresses are invalid?
I have second thoughts about swallowing instead of failing, it seems failing is safer and we avoid false positives, i.e. someone puts a bad IP, it listen in all interfaces, the test passes but we are not testing the IP that we want

Reply:
If all the addresses are invalid, we are gonna return an empty slice and we won't be listening on udp.
The test would fail, but I do agree, better be explicit. It would also make our life easier when triaging the failure.